com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDDefaultSettings.cs

```diff
@@ -0,0 +1,708 @@
+            assetCreated.lightLayerName2 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName2;
+            assetCreated.lightLayerName3 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName3;
+            assetCreated.lightLayerName4 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName4;
+            assetCreated.lightLayerName5 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName5;
+            assetCreated.lightLayerName6 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName6;
+            assetCreated.lightLayerName7 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName7;
+
+            assetCreated.decalLayerName0 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName0;
+            assetCreated.decalLayerName1 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName1;
+            assetCreated.decalLayerName2 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName2;
+            assetCreated.decalLayerName3 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName3;
+            assetCreated.decalLayerName4 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName4;
+            assetCreated.decalLayerName5 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName5;
+            assetCreated.decalLayerName6 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName6;
+            assetCreated.decalLayerName7 = oldAsset.currentPlatformRenderPipelineSettings.m_ObsoleteLightLayerName7;
+
+            assetCreated.shaderVariantLogLevel = oldAsset.m_ObsoleteShaderVariantLogLevel;
+            assetCreated.lensAttenuationMode = oldAsset.m_ObsoleteLensAttenuation;
+
+            System.Array.Resize(ref assetCreated.diffusionProfileSettingsList, oldAsset.m_ObsoleteDiffusionProfileSettingsList.Length);
+            for (int i = 0; i < oldAsset.m_ObsoleteDiffusionProfileSettingsList.Length; ++i)
+                assetCreated.diffusionProfileSettingsList[i] = oldAsset.m_ObsoleteDiffusionProfileSettingsList[i];
+
+            //3. Clear obsolete fields
+            if (bClearObsoleteFields)
+            {
+                oldAsset.m_ObsoleteDefaultVolumeProfile = null;
+                oldAsset.m_ObsoleteDefaultLookDevProfile = null;
+
+                oldAsset.m_ObsoleteRenderPipelineResources = null;
+                oldAsset.m_ObsoleteRenderPipelineRayTracingResources = null;
+
+                oldAsset.m_ObsoleteBeforeTransparentCustomPostProcesses = null;
+                oldAsset.m_ObsoleteBeforePostProcessCustomPostProcesses = null;
+                oldAsset.m_ObsoleteAfterPostProcessCustomPostProcesses = null;
+                oldAsset.m_ObsoleteBeforeTAACustomPostProcesses = null;
+                /* TODOJENNY - not sure why we cannot reset a string like that
```

Comment:
wouldn't be better to set these string to null (or leave them as is, I'm okay with both)